src/main/java/org/sagebionetworks/web/client/widget/entity/tabs/FilesTab.java

```diff
@@ -278,7 +273,12 @@ public void setTargetBundle(EntityBundle bundle, final Long versionNumber) {
     if (isCurrentVersion == null) {
       isCurrentVersion = true;
     }
-    tab.configureEntityActionController(bundle, isCurrentVersion, null);
+    tab.configureEntityActionController(
+      bundle,
+      isCurrentVersion,
+      null,
+      addToDownloadListWidget
+    );
```

Comment:
Need to pass along the `addToDownloadListWidget` so that the download menu controls the correct instance